Rakefile

```diff
@@ -14,7 +13,7 @@ WORKBENCH_DIR = FastlaneRake::WORKBENCH_DIR
 DOWNLOAD_DIR = FastlaneRake::DOWNLOAD_DIR
 DESTROOT = FastlaneRake::DESTROOT
 FASTLANE_GEM_VERSION = FastlaneRake::FASTLANE_GEM_VERSION
-XCODE_PATH = '/Applications/Xcode-7.3.1.app'
+RUBY__VERSION = FastlaneRake::RUBY__VERSION
```

Comment:
The double-underscore looks like a mistake. Intentional on `FastlaneRake` ? Also, doesn't extending bring in the constants already?

Reply:
Edit, I see the need for the double-under (`RUBY_VERSION` is already defined). Lets call it something else. `INSTALL_VERSION` or something.